src/py42/_internal/archive_access.py

```diff
@@ -81,12 +81,17 @@ def __init__(
         self._storage_archive_client = storage_archive_client
         self._restore_job_manager = restore_job_manager
 
-    def stream_from_backup(self, file_path):
-        metadata = self._get_file_via_walking_tree(file_path)
-        file_selection = self._build_file_selection(
-            metadata[u"path"], metadata[u"type"]
-        )
-        return self._restore_job_manager.get_stream(file_selection)
+    def stream_from_backup(self, file_path, exceptions):
+        if not isinstance(file_path, (list, tuple)):
+            file_path = [file_path]
+
+        file_selections = []
+        for path in file_path:
```

Comment:
can this be `file_paths`? The name `file_path` sounds like a single string.

Reply:
Yes, I can do that. This would be a little breaking change, for key-value parameters.